docs/modules/nifi/pages/index.adoc

```diff
@@ -1,8 +1,31 @@
 = Stackable Operator for Apache NiFi
+:description: The Stackable Operator for Apache NiFi is a Kubernetes operator that can manage Apache NiFi clusters. Learn about its features, resources, dependencies and demos, and see the list of supported NiFi versions.
+:keywords: k8s, Kubernetes, Stackable Operator, Apache NiFi, open source, operator, data science, data exploration, big data
 
-This is an operator for Kubernetes that can manage https://nifi.apache.org/[Apache NiFi] clusters.
+This Operator manages https://nifi.apache.org/[Apache NiFi] clusters on Kubernetes.
+Apache NiFi is an open-source data integration tool that provides a web-based interface for designing, monitoring, and managing data flows between various systems and devices, using a visual programming approach. It supports a wide range of data sources, formats, and features such as data provenance, security, and clustering.
 
-WARNING: This operator only works with images from the https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%2Fnifi[Stackable] repository
+== Getting started
+
+Get started with Apache NiFi and the Stackable Operator by following the xref:getting_started/index.adoc[] guide. It will guide you through the xref:getting_started/installation.adoc[installation] processs and xref:getting_started/first_steps.adoc[connect] to the NiFi webinterface. Afterwards have a look at the xref:usage_guide/index.adoc[] to learn how to configure your NiFi instance to your needs or run some <<demos, demos>> to learn more about using NiFi with other components.
```

Comment:
```suggestion
Get started with Apache NiFi and the Stackable Operator by following the xref:getting_started/index.adoc[] guide. It will guide you through the xref:getting_started/installation.adoc[installation] process and xref:getting_started/first_steps.adoc[connect] to the NiFi web interface. Afterwards have a look at the xref:usage_guide/index.adoc[] to learn how to configure your NiFi instance to your needs or run some <<demos, demos>> to learn more about using NiFi with other components.
```